facturx/facturx.py

```diff
@@ -147,3 +147,129 @@ def write_xml(self, path):
         with open(path, 'wb') as f:
             f.write(self.xml_str)
 
+    def __make_dict(self):
+        field_file = os.path.join(os.path.dirname(__file__), 'flavors\\fields.yml')
```

Comment:
you can import the variable `FIELDS` from the `flavors` module. That way we can later change the implementation of the fields settings.

Reply:
It will also save some code here and avoid repeating it 2x.

Reply:
Done